recipes/irida-sistr-results/build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# install dependencies through pip
+pip install -r requirements.txt
```

Comment:
Those should according bioconda's guidelines be added in the requirement section